.github/workflows/packages.yaml

```diff
@@ -0,0 +1,126 @@
+name: Build packages
+on: [push, pull_request, release]
```

Comment:
Keeping push and pull_request because it matches the previous behavior. But could consider limiting to release-only in the near future.

Reply:
If you remove this for `push`/`pull_request`, consider making it `manual_dispatch` as it'd come handy to build these manually if required during PRs.